browser-entry.js

```diff
@@ -52,6 +52,17 @@ process.removeListener = function(e, fn) {
   }
 };
 
+/**
+ * Implements listenerCount for 'uncaughtException'.
+ */
+
+process.listenerCount = function(name) {
```

Comment:
NOTE:  this is an observation and not a request for changes.

I don't love how `process` is shimmed here.  we might be able to get away with changing the prototype to the `EventEmitter` shim (loaded via browserify via [events](https://npm.im/events), because `process` should be an `EventEmitter`.

Reply:
Yeah I was surprised that that wasn't yet the case.